.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Test IPython beamline profiles
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  workflow_call:
+    inputs:
+      version: # the variable you can use in place of a matrix
+        required: true
+        type: string
+
+jobs:
+  testing_beamline_profiles:
+    name: ${{ matrix.repos.beamline-acronym }}-py${{ matrix.zenodo.python }}
+    strategy:
+      fail-fast: false
+      matrix:
+        repos:
+          - org: "NSLS2"
+            repo: "csx-profile-collection"
+            branch: "main"
```

Comment:
You may want to leave the branch empty - that way it will use the current ref for the PR/branch:
https://github.com/NSLS2/gha-beamline-integration-test/blob/1e4a95ba473c91fb199e7c155502868e56a8426b/action.yml#L59-L66

Reply:
Right that's a good point, thank you.